xray/backends/api.py

```diff
@@ -114,6 +114,9 @@ def open_dataset(filename_or_obj, group=None, decode_cf=True,
         used when reading data from netCDF files with the netcdf4 and h5netcdf
         engines to avoid issues with concurrent access when using dask's
         multithreaded backend.
+    drop_variables: iterable, optional 
```

Comment:
Maybe it would also be useful to support providing a single variable as a string? Someone is bound to try that. You can detect that with `isinstance(drop_variables, basestring)`.